pkg/codegen/internal/test/testdata/external-resource-schema/dotnet/ArgFunction.cs

```diff
@@ -13,6 +13,18 @@ public static class ArgFunction
     {
         public static Task<ArgFunctionResult> InvokeAsync(ArgFunctionArgs? args = null, InvokeOptions? options = null)
             => Pulumi.Deployment.Instance.InvokeAsync<ArgFunctionResult>("example::argFunction", args ?? new ArgFunctionArgs(), options.WithVersion());
+
+        public static Output<ArgFunctionResult> Invoke(ArgFunctionOutputArgs? args = null, InvokeOptions? options = null)
```

Comment:
It's called .Invoke but .InvokeOutput is also a possibility to match what we have in other languages. Votes?

Reply:
I'm leaning InvokeOutput too since in other langs we have _output etc, which Luke also preferred.

Reply:
Other languages have a very different invoke syntax so I'm not sure that should be a large factor. I expect we'd advise this new form by default - unless folks need the result for control flow, so having a simple unambiguous name would be my preference.

Resource constructors return outputs and they don't have `Output` in their name, so I don't see much difference here.

Reply:
FWIW, I don't feel strongly about this -- I could go either way. `Invoke` is the nicer name, but I do worry slightly that `Invoke` will be seen as the blocking variant of `InvokeAsync`, which isn't the case.